heartbeat/monitors/monitor.go

```diff
@@ -178,7 +181,8 @@ func newMonitorUnsafe(
 		// We need to use the lightweight wrapping for error jobs
 		// since browser wrapping won't write summaries, but the fake job here is
 		// effectively a lightweight job
-		wrappedJobs = wrappers.WrapLightweight(p.Jobs, m.stdFields, monitorstate.NewTracker(stateLoader, false))
+		m.stdFields.BadConfig = true
+		wrappedJobs = wrappers.WrapCommon(p.Jobs, m.stdFields, stateLoader)
```

Comment:
Small optimization, `max_attempts` should be overridden in case of bad config, or include a check on the retry logic. Atm, its retrying monitors with bad config